src/components/Hex/Hex.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+
+function Hex(props){
+    const [isCopied, setIsCopied] = useState(false);
+    const [textStyle, setTextStyle] = useState();
+
+    useEffect(() => {
+        console.log(isCopied);
```

Comment:
prefer to not have `console.log` littered! Or if this is intentional, prefer `console.info` to note that
```suggestion
```